src/Sulu/Bundle/AdminBundle/Resources/js/components/Switch/Switch.js

```diff
@@ -24,18 +25,29 @@ export default class Switch extends React.PureComponent<Props> {
     };
 
     render() {
-        const {className, type, name, checked, icon, children} = this.props;
+        const {
+            icon,
+            type,
+            name,
+            value,
+            checked,
+            useLabel,
+            children,
+            className,
+        } = this.props;
         const switchClass = classNames(
             switchStyles.switch,
-            className
+            className,
```

Comment:
According to mdn it is valid since the es2017 standard:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

eslint ignores trailing commas:
https://eslint.org/docs/rules/comma-dangle#options